src/app/pages/organization/companies/company/company.component.ts

```diff
@@ -57,7 +57,10 @@ export class CompanyComponent implements OnInit {
       this.router.navigate(['/']);
     }
     // get admin flag
-    this.isAdmin = this.authorizationService.isAdmin() || this.authorizationService.isSuperAdmin();
+    this.isAdmin = this.authorizationService.canCreateCompany() &&
+      this.authorizationService.canReadCompany() &&
+      this.authorizationService.canUpdateCompany() &&
+      this.authorizationService.canDeleteCompany();
```

Comment:
Nope, create member vars like canCreateCompany, canUpdateCompany...
And use them in the HTML template to show/hide control with *ngIf

Reply:
Thanks! updated